projects/plugins/jetpack/tests/php/media/test-class.jetpack-post-images.php

```diff
@@ -633,4 +633,53 @@ public function test_from_story_block_from_post_id_is_correct_array_videopress_w
 		$expected_poster_url = str_replace( 'mp4', 'jpg', $post_info['img_urls'][1] );
 		$this->assertEquals( $expected_poster_url, $images[1]['src'] );
 	}
+
+	/**
+	 * Test if the array extracted is empty in case post_id is null.
+	 *
+	 * @covers Jetpack_PostImages::from_gravatar
+	 * @since 11.3
```

Comment:
Jetpack includes tooling that allows you to make this dymamic. This way you do not have to know or think about when your change will be merged, you can let the tooling update that number for you in time with the next release.
https://github.com/Automattic/jetpack/blob/3a3c4adc1c0913bac224ddb832713bd6101fe1ab/projects/packages/README.md#L109-L116

```suggestion
	 * @since $$next-version$$
```

Reply:
Oh, that's good to know :)